src/devices/Mpu9250/README.md

```diff
@@ -0,0 +1,185 @@
+Console.WriteLine($"Mag Z = {magne.Z, 15}");
+```
+
+## Self-test
+
+A self-test is available for the gyroscope and the accelerometer.
+
+```csharp
+var resSelfTest = mpu9250.RunGyroscopeAccelerometerSelfTest();
+Console.WriteLine($"Self test:");
+Console.WriteLine($"Gyro X = {resSelfTest.Item1.X} vs >0.005");
+Console.WriteLine($"Gyro Y = {resSelfTest.Item1.Y} vs >0.005");
+Console.WriteLine($"Gyro Z = {resSelfTest.Item1.Z} vs >0.005");
+Console.WriteLine($"Acc X = {resSelfTest.Item2.X} vs >0.005 & <0.015");
+Console.WriteLine($"Acc Y = {resSelfTest.Item2.Y} vs >0.005 & <0.015");
+Console.WriteLine($"Acc Z = {resSelfTest.Item2.Z} vs >0.005 & <0.015");
+```
+
+The returned data are the raw data and allows you to estimate the quality of the test. The first item of the tuple is the gyroscope and the second one the accelerometer.
+
+No self-test is available for the magnetometer.
+
+## Calibration and bias
+
+You can calibrate the Gyroscope and the Accelerometer at the same time. This action is as well correcting the registers directly in the MPU9250 chip. Those data are lost in case of power stop but stays in case of soft reset.
+
+```csharp
+Console.WriteLine("Running Gyroscope and Accelerometer calibration");
+mpu9250.CalibrateGyroscopeAccelerometer();
+Console.WriteLine("Calibration results:");
+Console.WriteLine($"Gyro X bias = {mpu9250.GyroscopeBias.X}");
+Console.WriteLine($"Gyro Y bias = {mpu9250.GyroscopeBias.Y}");
+Console.WriteLine($"Gyro Z bias = {mpu9250.GyroscopeBias.Z}");
+Console.WriteLine($"Acc X bias = {mpu9250.AccelerometerBias.X}");
+Console.WriteLine($"Acc Y bias = {mpu9250.AccelerometerBias.Y}");
+Console.WriteLine($"Acc Z bias = {mpu9250.AccelerometerBias.Z}");
+```
+
+Calibration is as well available for the magnetometer (the AK8963). For this sensor, no correction is done.
+
+```csharp
+Console.WriteLine("Magnetometer calibration is taking couple of seconds, please be patient!");
+var mag = mpu9250.CalibrateMagnetometer();
+Console.WriteLine($"Bias:");
+Console.WriteLine($"Mag X = {mpu9250.MagnometerBias.X}");
+Console.WriteLine($"Mag Y = {mpu9250.MagnometerBias.Y}");
+Console.WriteLine($"Mag Z = {mpu9250.MagnometerBias.Z}");
+```
+
+## Units
+
+Al axis are oriented this way:
+            +Z   +Y
+          \  |  /
+           \ | /
+            \|/
+            /|\
+           / | \
+          /  |  \
+                 +X
+
+### Gyroscope
+
+The unit used for the gyroscope are degree per seconds.
+
+### Accelerometer
+
+The unit used for the accelerometer is G.
+
+### Magnetometer
+
+The unit used for the magnetometer is µTesla.
+
+### Temperature
+
+The Temperature is a normalized Units.Temperature which can provide Celsius, Kelvin or Fahrenheit degrees.
+
+## Measurement modes
+
+The MPU9250 offers a large variety of measurement modes. They can be changed and adjusted thru the properties like:
+
+* ```MagnetometerMeasurementMode``` to adjust the type of measurement for the magnetometer
```

Comment:
Why not single tick?

Reply:
if you mean "-" instead of "*", why not, so I changed.

Reply:
Not sure how to paste it in the markdown so will add extra spaces when needed:
Usually

```
` ` ` Text ` ` `
```

can be replaced with just

```
`Text`
```

and
```
` ` `
```

is just used for multi-line. Fine either way, I'm just not used to seeing triple backtick for single line and didn't even know this worked